Calculate Severity By Highest DBotScore - performance improvement (#30879)
demisto/content: Packs/CommonPlaybooks/ReleaseNotes/2_4_17.md

#### Playbooks

##### Calculate Severity By Highest DBotScore

> Improved the performance of the playbook by ensuring that only unique indicators are retrieved from the cache.

Reviewer: ```suggestion
Improved the performance of the playbook by ensuring that unique indicators were retrieved from the context.
```

Author: I'll keep it as is because we're grabbing them from the cache here, not from context :)
